tests/CMakeLists.txt

```diff
@@ -2,3 +2,4 @@ add_subdirectory(testmachinery)
 add_subdirectory(toyitems)
 add_subdirectory(model)
 add_subdirectory(viewmodel)
+add_subdirectory(sampleview)
```

Comment:
I suggest to move unit tests for sampleview in the subfolder of 'sampleview' example at next occasion.
I think DesignerScene will be unlikely the part of mvvm library.